src/lib.rs

```diff
@@ -36,3 +43,12 @@ mod transactions;
 mod tree_store;
 mod tuple_types;
 mod types;
+
+#[cfg(test)]
```

Comment:
@cberner Can I get your opinion on this (and 3891da5 in general)?

`tempfile` used as it was causes all tests to fail in WASI. This little workaround is needed in order to prevent that.

Is this an appropriate place for this function, or should it be placed elsewhere (or should I use a different solution entirely)? Wasn't 100% sure what your preference was, but wanted to get your feedback before I went all in and converted all tests to use this.

Reply:
I think I am instead just going to use `NamedTempFile::new_in("")` everywhere instead of `NamedTempFile::new()`. A bit easier that way, and tests still pass on other platforms. Only downside is CWD is used instead of whatever the system temp dir is, but I don't really think that matters too much for tests

Reply:
I'd rather use the helper function. The problem with cwd is that if the tests crash (or get ctrl-C) I think the temp files could end up leaking and polluting the working directory.

Reply:
Yea, I thought about & noticed that problem too. I might need to duplicate the helper function in a few places though (since tests/ isn’t apart of the crate), which is why I originally removed it after I added it.

A bit new to rust still, so I’ll just go ahead and duplicate the function in those several places it’s needed. If you know of a way to avoid the duplication though, I’m all ears